docs/aws.md

```diff
@@ -0,0 +1,123 @@
+# Statics and media files management with AWS
+
+We use AWS S3/CloudFront to store and distribute statics and media for the
+project in various environments. Buckets, distributions and IAM users creation
+is fully automated using [Terraform](https://www.terraform.io/).
+
+> ✋ If you plan to develop locally on the project, you don't have to configure
+> anything more. The following documentation targets operational users willing
+> to setup an AWS infrastructure to handle static and media files.
+
+## AWS Credentials
+
+To bootstrap the AWS infrastructure, you will need first to edit the `env.d/aws`
+environment file with your AWS credentials and settings. You can start by
+copying the `env.d/aws.dist` template _via_ the `make aws/env.d` command. Now
+edit this new file with your AWS secrets:
+
```

Comment:
make it clearer that this credentials should have sufficient access rights to : create S3 buckets, KMS keys, dynamoDB, IAM users and CloudFront distributions?